.github/workflows/ci.yml

```diff
@@ -153,149 +156,18 @@ jobs:
             }
           ]
       collection_pre_install: ''
-  splitter:
-    env:
-      source_dir: "./source"
-    runs-on: ubuntu-latest
-    outputs:
-      test_targets: ${{ steps.display.outputs.test_targets }}
-    steps:
-      - name: Checkout the collection repository
-        uses: actions/checkout@v3
-        with:
-          path: ${{ env.source_dir }}
-          fetch-depth: "0"
-
-      - name: list changes for pull request
-        id: splitter
-        uses: ansible-network/github_actions/.github/actions/ansible_test_splitter@main
-        with:
-          collections_to_test: ${{ env.source_dir }}
-          total_jobs: 8
-
-      - name: display targets
-        id: display
-        run: echo "test_targets=${{ steps.splitter.outputs.test_targets }}" >> $GITHUB_OUTPUT
-        shell: bash
-
   integration:
+    uses: guidograzioli/kubernetes.kubevirt/.github/workflows/integration.yml@integration_tests
```

Comment:
No this needs to be change to `kubevirt/kubernetes.kubevirt`.

Reply:
Correct; but because the action is not retrievable from actions while it exists in a PR only, I am pointing to my fork to just see the tests run and go green; then we can amend to the final location before merging

Reply:
This rest of the PR `lgtm`, can you change this now or do you want to line up a follow up PR?